lib/android-helpers.js

```diff
@@ -335,6 +335,14 @@ helpers.initUnicodeKeyboard = async function (adb) {
   return defaultIME;
 };
 
+helpers.setMockLocationApp = async function (adb, app) {
+  if (await adb.getApiLevel() < 23) {
```

Comment:
I also just realized, `getApiLevel` presumably returns a string? At least, looking in the code, I couldn't see us doing any conversion here. Maybe to have a safer comparison, we should convert this to a number:

```js
if (parseInt(await adb.getApiLevel(), 10) < 23) {
```

alternatively, we could update the library method to return a number itself; not sure if that will break how anyone else is depending on the API already, though.